src/OwaspHeaders.Core.csproj

```
@@ -3,21 +3,28 @@
     <Description>An ASP.NET Core Middleware which adds the OWASP recommended HTTP headers for enhanced security.</Description>
     <VersionPrefix>8.1.0</VersionPrefix>
     <Authors>Jamie Taylor</Authors>
+    <PackageProjectUrl>https://ScottPlot.NET</PackageProjectUrl>
```

Please could you change this URL to the one below?

```suggestion
    <PackageProjectUrl>[https://ScottPlot.NET](https://github.com/GaProgMan/OwaspHeaders.Core)</PackageProjectUrl>
```

I appreciate that I don't have a public URL for the package and that you wouldn't have known that ahead of time. I have a TODO for creating a github io page for the project and adding some documentation to it. Once I've done that, I'll make a change to this element, pointing it to the gituhb io site.